apps/condo/domains/common/utils/serverSideAddressApi.js

```diff
@@ -4,11 +4,12 @@ const fetch = require('node-fetch')
 const conf = require('@open-condo/config')
 
 const FAKE_SUGGESTIONS = (conf['FAKE_ADDRESS_SUGGESTIONS'] && conf['FAKE_ADDRESS_SUGGESTIONS'] === 'true') || false
+const IS_BUILD = conf['DATABASE_URL'] === 'undefined'
```

Comment:
Wow! We have a chance to eliminate `PHASE=build` :)

Reply:
We don't have this variable ap build time